src/razor/src/dynamicFile/dynamicFileInfoHandler.ts

```diff
@@ -99,22 +92,25 @@ export class DynamicFileInfoHandler {
             if (this.documentManager.isRazorDocumentOpenInCSharpWorkspace(vscodeUri)) {
                 // Open documents have didOpen/didChange to update the csharp buffer. Razor
                 // does not send edits and instead lets vscode handle them.
-                return new ProvideDynamicFileResponse(
-                    { uri: virtualCsharpUri },
-                    null,
-                    csharpDocument.checksum,
-                    csharpDocument.checksumAlgorithm,
-                    csharpDocument.encodingCodePage
-                );
+                // return new ProvideDynamicFileResponse(
```

Comment:
Intentional?

Reply:
Nope. I need to double check this is correct. prior to this I think it was unnecessary to send this and null should be fine. Left a commented out to remind myself then promptly forgot